src/NEAR/Serialization.cpp

```diff
@@ -68,6 +118,21 @@ static void writeAction(Data& data, const Proto::Action& action) {
         case Proto::Action::kFunctionCall:
             writeFunctionCall(data, action.function_call());
             return;
+        case Proto::Action::kStake:
+            writeStake(data, action.stake());
+            return;
+        case Proto::Action::kAddKey:
+            writeAddKey(data, action.add_key());
+            break;
```

Comment:
`return` and `break` usage are mixed in this `switch`, can you consolidate them?